staging/src/k8s.io/apiextensions-apiserver/go.mod

```diff
@@ -34,6 +34,7 @@ require (
 )
 
 replace (
+	github.com/pkg/errors => github.com/pkg/errors v0.8.0
```

Comment:
update the preferred version (v0.8.1) using `hack/pin-dependency.sh` and `hack/update-vendor.sh` rather than pinning to an older version like this

Reply:
thanks @liggitt .  `hack/pin-dependency.sh` is so cool. PR updated.